### ISSUE TITLE
Forgotten Password Page

### ISSUE THREAD

**Issue by ahansson89** [AnomalyInnovations/serverless-stack-com#142]
I would love to see a forgotten password method.

**Comment by jayair** [comment]
@ahansson89 Yeah I might add it as an "Extra Credit" chapter. It's weird that these are not well documented on the AWS side.
